.azure-devops/cleanup.yaml

```diff
@@ -0,0 +1,3 @@
+steps:
+  - task: mspremier.PostBuildCleanup.PostBuildCleanup-task.PostBuildCleanup@3
```

Comment:
What is the purpose of the existence of this file? The https://github.com/OfficeDev/Office-Addin-Scripts/pull/530 didn't have this

Reply:
This specific to the Mac case since we are re-using the machine and not just dumping a VM.  Office-Addin-Scripts does have the file as well.